crates/swc_ecma_minifier/src/compress/optimize/iife.rs

```diff
@@ -241,6 +242,7 @@ where
                                     }
 
                                     match &*arg.expr {
+                                        Expr::Lit(Lit::Regex(..)) => false,
```

Comment:
Can you add a test case for this? I initially thought that it should return true, but the regex literal should be pure so it can be dropped.

Reply:
Oh you are right, I'll revert this change

Reply:
Thank you so much